test_vm/tests/batch_onboarding_deals_test.rs

```diff
@@ -8,3 +10,10 @@ fn batch_onboarding_deals() {
     let v = TestVM::<MemoryBlockstore>::new_with_singletons(&store);
     batch_onboarding_deals_test(&v);
 }
+
+#[test]
+fn pre_commit_requires_commd() {
+    let store = MemoryBlockstore::new();
+    let v = TestVM::<MemoryBlockstore>::new_with_singletons(&store);
+    pre_commit_requires_commd_internal(&v);
```

Comment:
The established pattern is to suffix `_test` for the integration test implementations.